.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Question
+about: Question relating to Kosmos.
+labels: kind/question
+
+---
+
+<!-- Please use this template while asking a new question. Thanks!
+Please make sure you have read the FAQ and searched the issue list.
+- FAQ: https://github.com/Kosmos-io/Kosmos/blob/master/docs/frequently-asked-questions.md
+- Issues: https://github.com/Kosmos-io/Kosmos/issues?q=is%3Aissue+
```

Comment:
https://github.com/kosmos-io/kosmos/issues

Reply:
ok. I will update it as soon as possible